### ISSUE TITLE
Stop adding a line break

### ISSUE THREAD

**Issue by superbrothers** [itaysk/kubectl-neat#56]
This useful command always adds a new line break to the last line of the manifest. This is superfluous and I want to remove it.

```
$ git show -q
commit cb318129e812c819a80199febc38e5a18e6b64f0 (HEAD -> master, origin/master, origin/HEAD)
Author: karancode <karan.thanvi@paypay-corp.co.jp>
Date:   Tue Oct 6 18:41:08 2020 +0900

    update ci workflow
$ cat <<EOL | kubectl-neat
apiVersion: apps/v1
kind: Deployment
metadata:
  creationTimestamp: null
  labels:
    app: nginx
  name: nginx
spec:
  replicas: 1
  selector:
    matchLabels:
      app: nginx
  strategy: {}
  template:
    metadata:
      creationTimestamp: null
      labels:
        app: nginx
    spec:
      containers:
      - image: nginx
        name: nginx
        resources: {}
status: {}
EOL
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: nginx
  name: nginx
spec:
  replicas: 1
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      creationTimestamp: null
      labels:
        app: nginx
    spec:
      containers:
      - image: nginx
        name: nginx

```

**Comment by superbrothers** [comment]
@itaysk Would you please create a new release? I'd like to use #43 and #57.

**Comment by itaysk** [comment]
@superbrothers yes I've just released v2.0.2
